lib/player/player.go

```diff
@@ -429,6 +453,38 @@ func (p *Player) LastPlayed() int64 {
 	return p.lastPlayed.Load()
 }
 
+// translateEvent translates events if applicable and return if they should be
+// skipped.
+func (p *Player) translateEvent(evt events.AuditEvent) (events.AuditEvent, bool) {
```

Comment:
Nit: Named return args would be helpful here to help explain what the bool argument is.
```suggestion
func (p *Player) translateEvent(evt events.AuditEvent) (translatedEvent events.AuditEvent, shouldSkip bool) {
```